docs/0.4.0-alpha/basic_syntax/variables.md

```diff
@@ -27,6 +29,11 @@ let result = 123
 let result = "Hello my friend"
 ```
 
-## Reserved Prefix
+# Constant
+
+Constant is a type of variable that cannot be modified.
 
-The Amber compiler reserves all identifiers starting with double underscore `__` in addition to keywords like `let`, `if`, etc.
+```ab
+const gravity = 9.81
```

Comment:
I love this example!  But if you really wanted a universal constant, you could use `speed_of_light`.

Reply:
Good idea!